stable/pachyderm/README.md

```diff
@@ -0,0 +1,142 @@
+Pachyderm Helm Chart
+====================
+
+Pachyderm is a language-agnostic and cloud infrastructure-agnostic large-scale data processing framework based on software containers. This chart can be used to deploy Pachyderm backed by object stores of different Cloud providers.
+
+* https://pachyderm.io
+* https://github.com/pachyderm/pachyderm
+
+
+Prerequisites Details
+---------------------
+
+-	Dynamic provisioning of PVs (for non-local deployments)
+
+General chart settings
+----------------------
+
+The following table lists the configurable parameters of `pachd` and their default values:
+
+| Parameter                | Description           | Default           |
+|--------------------------|-----------------------|-------------------|
+| `pachd.image.repository` | Container image name  | `pachyderm/pachd` |
+| `*.image.tag`            | Container image tag   | `1.6.6`           |
```

Comment:
you could use something like `<latest version>` to avoid having to update this README each time the version is bumped